### PR TITLE
Create new changelog section following release

### PR DESCRIPTION
**Pull request by kjun9**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,17 @@
...
 
-[Full Changelog](https://github.com/stellargraph/stellargraph/compare/HEAD...v0.8.2)
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.9.0...HEAD)
+
+### Major features and improvements
+
+### Breaking changes
+
+### Experimental features
+
```
Comment (huonw) [inline CHANGELOG.md]: We can potentially pre-fill the paragraph describing experimental features here:

```suggestion

Some new algorithms and features are still under active development, and are available as an experimental preview. However, they may not be easy to use: their documentation or testing may be incomplete, and they may change dramatically from release to release. The experimental status is noted in the documentation and at runtime via prominent warnings.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,7 +2,17 @@
 
 ## [HEAD](https://github.com/stellargraph/stellargraph/tree/HEAD)
 
-[Full Changelog](https://github.com/stellargraph/stellargraph/compare/HEAD...v0.8.2)
+[Full Changelog](https://github.com/stellargraph/stellargraph/compare/v0.9.0...HEAD)
+
+### Major features and improvements
+
```
Comment (huonw) [inline CHANGELOG.md]: The suggestion on #728 was that it's worth having a section for new algorithms:

```suggestion

- New algorithms:
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,7 +2,19 @@
...
+- New algorithms:
```
Comment (kjun9) [inline CHANGELOG.md]:
```suggestion
- New algorithms:

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,7 +2,19 @@
...
+### Breaking changes
+
+### Experimental features
+
+Some new algorithms and features are still under active development, and are available as an experimental preview. However, they may not be easy to use: their documentation or testing may be incomplete, and they may change dramatically from release to release. The experimental status is noted in the documentation and at runtime via prominent warnings.
```
Comment (kjun9) [inline CHANGELOG.md]:
```suggestion
Some new algorithms and features are still under active development, and are available as an experimental preview. However, they may not be easy to use: their documentation or testing may be incomplete, and they may change dramatically from release to release. The experimental status is noted in the documentation and at runtime via prominent warnings.

```

**Pull request merged**